roles/cfg_openwrt/templates/corerouter/config/dhcp.j2

```diff
@@ -13,6 +13,7 @@ config dnsmasq
 	option noping '{{ dhcp_no_ping|int if dhcp_no_ping is defined else 1 }}'
 	option quietdhcp '1'
 	option cachesize '{{ dns_cache_size | default(10000) }}'
+ 	list server '/olsr/'
```

Comment:
Can you explain the config syntax? I don't understand why this config option should exclude olsr domains from a lookup.

Reply:
I found it: https://openwrt.org/docs/guide-user/base-system/dhcp_configuration#selective_dns_forwarding

Probably you could also do `list server '/olsr/127.0.0.1'`

Reply:
> Probably you could also do list server '/olsr/127.0.0.1'

That *might* do a loop, `/olsr/` simply means that *.olsr lookups are forwarded to a "null server" if we don't already know the answer, i.e. they're not forwarded at all.

Reply:
If OLSRd served a location DNS server on e.g. port 10053 instead of writing files, we could set it to `/olsr/127.0.0.1#10051` instead of refreshing files.

Reply:
> If OLSRd served a local DNS server on e.g. port 10053 instead of writing files, we could set it to `/olsr/127.0.0.1#10051` instead of refreshing files.

That would probably solve https://github.com/OLSR/olsrd/issues/116 along the way.